src/server.ts

```diff
@@ -67,9 +67,12 @@ const start = async (): Promise<FastifyInstance> => {
   );
 
   try {
-    const PORT = envVariablesConfig.SERVER_PORT;
+    const PORT = Number(envVariablesConfig.SERVER_PORT) ?? 8003;
```

Comment:
Do not set default values for these things, throw an error if the variable is not present.